src/ActionModal/utils/MessageConstructor.js

```diff
@@ -62,174 +56,31 @@ export const getMultiMessage = async (context, messages) => {
   }
 }
 
-// Bank
 /* istanbul ignore next */
-export function MsgSend(
-  senderAddress,
-  {
-    toAddress,
-    amounts // [{ denom, amount}]
-  }
-) {
-  return {
-    type: `cosmos-sdk/MsgSend`,
-    value: {
-      from_address: senderAddress,
-      to_address: toAddress,
-      amount: amounts.map(Coin)
-    }
-  }
-}
+async function getMessageFormatter(network, messageType) {
+  let networkMessages
+  try {
+    networkMessages = await import(`./networkMessages/${network}.js`)
+  } catch (err) {
+    throw new Error("Signing for this network is not enabled.")
```

Comment:
this error seems like it's for a user - but could a user encounter it? feels like they shouldn't get to the point where we're requesting networkMessage schema if transactions aren't enabled.

Reply:
True! This is not only for the user but also a good message for us developers. :)